### PR TITLE
[HOLD] Add support for datasetless operator

### PR DESCRIPTION
**Pull request by imanjra**
**Todo: add execution time protection to prevent execution**

## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: fiftyone/operators/operator.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class OperatorConfig(object):
             when app is in the light mode
         dark_icon (None): icon to show for the operator in the Operator Browser
             when app is in the dark mode
+        datasetless (False): whether the operator can be executed without dataset
```
Comment (ritch) [inline fiftyone/operators/operator.py:35]: This needs more clarification

**Review by ritch — CHANGES_REQUESTED** [review]
All looks good implementation wise. The doc could use some additional explanation.

**Comment by brimoor** [comment]
Per offline discussion, decided to hold off on this for now until we consider other alternatives like `resolve_listing()`.

**Comment by imanjra** [comment]
Closing for now

**Pull request closed without merging**